tailwind.config.js

```diff
@@ -50,13 +53,55 @@ export default {
           '0%': { opacity: '1' },
           '100%': { opacity: '0' },
         },
+        walletContainerIn: {
+          '0%': { opacity: '0', transform: 'translateY(-10px)' },
+          '100%': { opacity: '1', transform: 'translateY(0)' },
+        },
+        walletContainerOut: {
+          '0%': { opacity: '1', transform: 'translateY(0)' },
+          '100%': { opacity: '0', transform: 'translateY(-10px)' },
+        },
+        walletIslandContainerItemIn: {
+          '0%': { opacity: '0', transform: 'translateY(10px)' },
+          '100%': { opacity: '1', transform: 'translateY(0)' },
+        },
```

Comment:
Since they are specific to `Wallet` and `WalletIsland`, should these configs live elsewhere?

Reply:
i actually think i was overly specific about the name here, i think we should remove `wallet` and `walletIsland` from all the names, as they can be used for any container / container items.